### ISSUE TITLE
clusterProfile: :compareCluster fails with error ‘'package' must be of length 1

### ISSUE THREAD

**Issue by charlesgwellem** [GuangchuangYu/clusterProfiler#195]
Is it possible to perform gene cluster comparison analysis with `clusterProfiler::compareCluster` using the `enrichPathway `function with a mouse data set?
This is what I havce done: 

```
require(clusterProfiler)

# just a representative example of my mouse gene set
my_genes <- list(wt =c( "1", "100", "1000", "10000"), wt1 = c("100000", "1000000", "10000000", "100000000"), wt3 = c("2", "200", "2000"), wt4 = c("3", "300"))
```
I want to run `compareCluster `function on `my_genes `  with the `enrichPathway `function list above: I presume that the  default the organism is human for  `compareCluster ` but I have a mouse data set so I did:

`res <- compareCluster(my_genes , fun="enrichPathway", organism = "mmu")`

I got the error error ‘'package' must be of length 1 and my R session shut down. 

I will be very grateful for any suggestions on how I can run the clusterProfiler::compareCluster` using the `enrichPathway `function with my mouse data set

**Comment by russHyde** [comment]
Hi. If you look at the help page for the function `enrichPathway` (which is defined in `reactomePA`; https://bioconductor.org/packages/release/bioc/manuals/ReactomePA/man/ReactomePA.pdf), the valid values for the `organism` argument are one of "human", "rat", "mouse", "celegans", "yeast", "zebrafish", "fly". 

When you call compareCluster with fun="enrichPathway", all arguments to `compareCluster` other than `geneClusters`, `fun` and `data` are passed to the enrichPathway function (so they should match the values expected by enrichPathway).

TLDR: if you change "mmu" to "mouse" your call will work
